apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -177,6 +177,10 @@ type ScaledObjectStatus struct {
 	PausedReplicaCount *int32 `json:"pausedReplicaCount,omitempty"`
 	// +optional
 	HpaName string `json:"hpaName,omitempty"`
+	// +optional
+	TriggersNames *string `json:"triggersNames,omitempty"`
+	// +optional
+	AuthenticationsNames *string `json:"authenticationsNames,omitempty"`
```

Comment:
I think it's good to leave Triggers in printcolumn but keep TriggersNames in object as other place Triggers array means contains Trigger object instead of just Trigger names.

Reply:
maybe `triggerType`? Because we also have `trigger.name`

Reply:
The same for `authentication`, `authenticationName` is a bit weird, I would use just `authentication` or `authenticationType`